test/Microsoft.TestPlatform.AcceptanceTests/IsExternalInit.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !NET5_0_OR_GREATER
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Runtime.CompilerServices;
+
+/// <summary>
+/// Reserved for use by the compiler for records.
+/// /!\ Do not use directly. /!\
+/// </summary>
+[ExcludeFromCodeCoverage, DebuggerNonUserCode]
+internal static class IsExternalInit
```

Comment:
Ideally we would want this file to be added to all projects but keeping that for later.